public/app/plugins/datasource/cloudwatch/module.tsx

```diff
@@ -0,0 +1,13 @@
+import { DataSourcePlugin } from '@grafana/data';
+import { CloudWatchQueryCtrl } from './query_ctrl';
+import Datasource from './datasource';
+import { ConfigEditor } from './components/ConfigEditor';
+
+class CloudWatchAnnotationsQueryCtrl {
+  static templateUrl = 'partials/annotations.editor.html';
+}
+
+export const plugin = new DataSourcePlugin(Datasource)
```

Comment:
The DataSourcePlugin needs to be typed with its options...
see https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/input/module.ts#L9